components/Button/StartGame.js

```diff
@@ -0,0 +1,57 @@
+import Image from "next/image";
+import gummiBearPicture from "../../images/altair-valasek-6uHnajYSsW0-unsplash.jpg";
+import styled from "styled-components";
+export default function StartGame({ setGameStarted, wonGame, setWonGame }) {
+  function startTheGame() {
+    setGameStarted(true);
+    setWonGame(false);
+  }
+  return wonGame ? (
+    <>
+      <StyledHeader>
+        Du hast das Wort erraten!!! Hier sind ein paar Gummibärchen als
+        Belohnung...
+      </StyledHeader>
+      <Image
+        src={gummiBearPicture}
+        alt="a picture of gummi bears"
+        width="200"
```

Comment:
I´m not sure, does this work without a specification like px?

Reply:
Yep, the next/image width and height is by default set as px :)